TLBC2App/Db/ADTLBC2.template

```diff
@@ -28,19 +28,23 @@ record("*", "$(P)$(R)Gain") {
 }
 
 record("*", "$(P)$(R)MinX") {
-    field(PINI, "NO")
+    field(PINI, "YES")
+    field(PHAS, 2)
 }
 
 record("*", "$(P)$(R)MinX") {
```

Comment:
It is missing (in another commit?) this typo fix: `s/MinX/MinY/`.

Reply:
I fixed this locally; will request write permissions to the repo (lost after the transfer) to be able to finish it.